DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/basic_triggers/test_interfaces_relaod.py

```diff
@@ -46,15 +47,17 @@ async def test_reload_interface(testbed):
         )
         fname = "interfaces." + dev.device_id
         old_fname = "interfaces." + dev.device_id + ".old"
-        fp = open(fname, "w")
+        # copy the original file
+        device.applog.info("Getting the original interface file")
+        await device.scp(old_fname, "/etc/network/interfaces", remote_to_local=True, sudo=True)
+        # copy the original file.
+        shutil.copy(fname, old_fname)
```

Comment:
@muchetan You are replacing the recently copied file from the remote. Is this correct?